cmd/uninstall.go

```diff
@@ -16,15 +16,24 @@ var uninstallCmd = &cobra.Command{
 	Short: "Uninstall KubeArmor from a Kubernetes Cluster",
 	Long:  `Uninstall KubeArmor from a Kubernetes Clusters`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		err := install.K8sUninstaller(client, uninstallOptions)
-		return err
+		if uninstallOptions.Legacy {
+			if err := install.K8sLegacyUninstaller(client, uninstallOptions); err != nil {
+				return err
+			}
+		} else {
+			if err := install.K8sUninstaller(client, uninstallOptions); err != nil {
+				return err
+			}
+		}
+		return nil
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(uninstallCmd)
 
-	uninstallCmd.Flags().StringVarP(&uninstallOptions.Namespace, "namespace", "n", "", "If no namespace is specified, it defaults to all namespaces and deletes all KubeArmor objects across them.")
+	uninstallCmd.Flags().StringVarP(&uninstallOptions.Namespace, "namespace", "n", "kubearmor", "If no namespace is specified, it defaults to all namespaces and deletes all KubeArmor objects across them.")
```

Comment:
uninstall is meant to be namespace agnostic. It will be responsibility of the client to find which namespace the installation is in provided namespace is not provided.

Reply:
Done.